product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx

```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
 navTitle: "Create an account" 
 ---
 
-Registering an account on [enterprisedb.com](https://www.enterprisedb.com/) gets you access to the BigAnimal trial, as well as EDB's training and software downloads. 
+You will need to register with EDB in order to access BigAnimal free trial, as well as EDB's training and software downloads.
 Accounts are free, and registration takes less than a minute.
 
 1. Navigate to <https://biganimal.com>.
-2. Select the **Try it for free** button.
-3. Fill in the form, and agree to the terms and conditions.
+2. Select the **Test drive for free** button.
+3. Fill in the form, and agree to the **Terms and Conditions** and the **Privacy Policy**.
 4. Select **Submit**.
 5. Check your email. You should receive an email from EDB containing an activation link.
-6. Select the activation link and enter a password for your EDB account.
+6. Click the activation link and enter a password for your EDB account.
 
 You're then redirected to the BigAnimal portal.
 
 !!! Note "Already have an EDB account?"
-    If you've already signed up for an EDB account, you can navigate to <https://portal.biganimal.com/> to log in.
+    If you've already signed up for an EDB account, you can navigate to <https://freetrial.biganimal.com/> to log in.
+    Please continue to use this portal in order to access BigAnimal for the duration of your free trial.
```

Comment:
If you already signed up for an EDB account, navigate to <https://freetrial.biganimal.com/>  to log in. Continue to use this portal to access BigAnimal for the rest of your free trial.